index.html

```diff
@@ -1,189 +1,294 @@
-        <h2 class="h1_contact">Contact me!</h2>
-        <p class="p_contact">
-          If you have an application you are intrested in developing, a feature
-          that you need built or a project that needs coding. I'd love to help
-          with it
+      </li>
+    </ul>
+  </section>
+
+  <!--ABOUT START-->
+  <section class="about">
+    <div class="about-info">
+      <div class="about-info-text">
+        <h2 class="h2-1">About<br />Myself</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
         </p>
-        <form
-          class="forms-input"
-          action="https://formspree.io/f/mayajwnb"
-          method="post"
-        >
-          <input
-            type="text"
-            name="name"
-            class="input"
-            maxlength="30"
-            required
-            placeholder="Full Name"
-          />
-          <input
-            name="email"
-            type="email"
-            class="input"
-            required
-            placeholder="Email"
-          />
-          <input
-            name="message"
-            type="text"
-            class="msg"
-            maxlength="500"
-            required
-            placeholder="Write your message here"
-          />
-          <button type="submit" class="contact-button">Get It touch</button>
-        </form>
       </div>
+      <div class="social-connect">
+        <h3>LET’S CONNECT</h3>
+        <ul class="social-links">
+          <li>
+            <a href="#">
+              <img src="IMG/twiterIcon.svg" alt="twiter-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="IMG/linkedinIcon.svg" alt="linked-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="IMG/mIcon.svg" alt="m-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="IMG/gitHubIcon.svg" alt="github-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="IMG/redIcon.svg" alt="red-icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+      <a class="get-resume" href="#"> Get my resume </a>
+    </div>
+    <div class="about-skills">
+      <ul class="skills">
+        <li class="skill active-skill">
+          <div class="skill-name">
+            <p class="name">Langguages</p>
+            <img src="IMG/arrDown.png" alt="arrDown" />
+          </div>
+          <ul class="inner-skills">
+            <li><img src="IMG/jsLogo.png" alt="jsLogo" />JavaScript</li>
+            <li><img src="IMG/htmlLogo.png" alt="htmlLogo" />HTML</li>
+            <li><img src="IMG/cssLogo.png" alt="cssLogo" />CSS</li>
```

Comment:
- [ ] This CSS image is overflowing in the desktop version. See in the screenshot below.

![Screenshot from 2021-06-11 16-44-05](https://user-images.githubusercontent.com/22434660/121678216-8baa3f80-cad4-11eb-901f-9256b7d73aa6.png)